interfaces/builtin/bluez.go

```diff
@@ -91,7 +96,7 @@ var bluezConnectedPlugAppArmor = []byte(`
 # Allow all access to bluez service
 dbus (receive, send)
     bus=system
-    peer=(label=bluez5_bluez_*),
+    peer=(label=snap.bluez.*),
```

Comment:
This is hard-coding an interface to be used by a specifically named snap, which doesn't seem to make sense for an interface in a platform.

@zyga can you please work with him to ensure we have the proper content in this template?  I thought we had already discussed this last week?